### PR TITLE
Add initial config docs for b2b

### PR DESCRIPTION
**Pull request by shaunmaharaj**
Description:

- Configurations
- Note to direct to b2b docs

Linting:

- [x] No linting errors

Tests:

- [x] Smoke tests (mvn clean install -Dcucumber.options="--tags @smoketest" from test)
- [ ] Manual tests

Documentation:

- [x] Requires documentation updates

### CODE REVIEW

**Comment by shaunmaharaj** [comment]
@asmurthwaite Thanks for the review. I've addressed all review comments. If there are no further comments then I'll merge.

**Comment by shaunmaharaj** [comment]
> The only comment that I have at this time is with regards to the use of EAM in the docs as I know it's changing. I'm not sure if it's finalized @shaunmaharaj .

Yep. I'll do a mass rename as necessary

File: README.md [hunk 1]
```diff
@@ -79,6 +79,12 @@ You must configure the following parameters in the `./src/ep.config.json` file:
 |`siteImagesUrl`| Optional| String| The path to the site content and marketing images hosted on an external CMS. Set this parameter to the complete URL of the images by replacing the filename and file extension with `%fileName%`. This parameter is populated with the values set in the components when the page is loaded, and uses the assets locally available in the `./src/images/site-images` directory.|
 |`arKit.enable`| Optional| Boolean| Enable elements for ARKit's Quick Look capability to load on a product display page. When `arKit.enable` is enabled, any product images that have hosted ARKit USDZ files are wrapped with an anchor tag referencing the file hosted on an external CMS.|
 |`arKit.skuArImagesUrl`| Optional| String| The path to the USDZ files hosted on an external CMS used for ARKit Quick Look. Set this parameter to the complete URL of the files by replacing the `sku/file-name` with `%sku%`. This value is populated when the page is loaded with values retrieved by Cortex.|
+|`b2b.enable`| Optional| Boolean| The general configuration to enable the B2B e-commerce shopping flow components in the storefront.|
```
Comment (sminnath) [inline README.md]: ecommerce

File: README.md [hunk 2]
```diff
@@ -79,6 +79,12 @@ You must configure the following parameters in the `./src/ep.config.json` file:
...
+|`b2b.eamAPI.pathForProxy`| Optional| String| The path to which the [Webpack](https://webpack.js.org/) proxy uses to route Cortex calls from the storefront to the publically hosted EAM service. This value is a URL consisting of a hostname and port of a running Cortex for EAM instance. Leave this field blank to disable proxy.|
```
Comment (sminnath) [inline README.md]: The path to which the: **This sounds wrong ** 

you mean?
The path that the [Webpack](https://webpack.js.org/) proxy uses to route Cortex calls from the storefront to the publicly hosted Account Management service. This value is a URL that consists of the hostname and port of a running Cortex for an Account Management instance. Leave this field blank to disable proxy.

spell error-> publicly

File: README.md [hunk 3]
```diff
@@ -79,6 +79,12 @@ You must configure the following parameters in the `./src/ep.config.json` file:
...
+|`b2b.keyCloak.callbackUrl`| Optional| String| The URL that is passed to KeyCloak, and redirected to upon a successful login.|
```
Comment (sminnath) [inline README.md]: The URL that is passed to KeyCloak (for what) and redirected to upon a successful login(or authentication)?.

File: README.md [hunk 4]
```diff
@@ -79,6 +79,12 @@ You must configure the following parameters in the `./src/ep.config.json` file:
...
+|`b2b.keyCloak.loginRedirectUrl`| Optional| String| The KeyCloak log on URL.|
```
Comment (sminnath) [inline README.md]: log on URL?

File: README.md [hunk 5]
```diff
@@ -79,6 +79,12 @@ You must configure the following parameters in the `./src/ep.config.json` file:
...
+|`b2b.keyCloak.logoutRedirectUrl`| Optional| String| The KeyCloak log off URL.|
```
Comment (sminnath) [inline README.md]: log off URL?

**Comment by rmkc** [comment]
Just a global comment:

Since the audience is the developer, this level of brevity in API documentation is very typical, so the language and style used I am very much ok with.  

The focus should be on spelling, obvious grammar errors, inconsistency in use. In fact, a linting test could automate much of this to suit the EP style guide (noun - logon / logoff vs verbs - log on or log off.)

so "successful login" because it is an action becomes "successful log in"
and "KeyCloak log on URL" becomes "KeyCloak logon URL"

When discussing URLs, it may be useful to include Examples: followed by the expected format, say: `"https://123mysite.com:123/path1/path2/file.html"`  to eliminate any confusion, or force people to go elsewhere to precisely understand what is asked for.

**Pull request merged**